ament_index_cpp/CMakeLists.txt

```diff
@@ -38,6 +41,7 @@ install(DIRECTORY include/
   DESTINATION include)
 
 install(TARGETS ${PROJECT_NAME}
+  EXPORT export_${PROJECT_NAME}
```

Comment:
This install rule is being "identified" by the name `export_${PROJECT_NAME}`. `ament_export_interfaces(<same_identifier>)` creates the necessary part for the CMake config file that the installed targets can be imported in downstream packages.